nym-node-status-api/Cargo.toml

```diff
@@ -23,7 +23,8 @@ futures-util = { workspace = true }
 moka = { workspace = true, features = ["future"] }
 nym-bin-common = { path = "../common/bin-common", features = ["models"]}
 nym-common-models = { path = "../common/models" }
-nym-explorer-client = { path = "../explorer-api/explorer-client" }
+# nym-explorer-client = { path = "../explorer-api/explorer-client" }
+nym-explorer-client = { git = "https://github.com/nymtech/nym", branch = "release/2024.13-magura" }
```

Comment:
I assume these git dependencies into it's own repo is a temporary thing until we make it all work?

Reply:
oh yeah. since we're merging it into monorepo and we actually did update NS now, can we point to the local code instead? (I can resync develop with magura as I've done yesterday so that it contains the new endpoints we have just added)

Reply:
done